server/routes/login.js

```diff
@@ -129,8 +125,12 @@ module.exports = () => (router) => {
       }
 
       const quantumAddresses = config.quantumAddresses.split(',')
-
-      if (config.twoFactorAuthOn === 'true' && ipRangeCheck(ipAddress, quantumAddresses) === false) {
+      const hmppsAuthMFAUser = jwtDecode(req.user.token).authorities.includes('ROLE_MFA')
+      if (
+        hmppsAuthMFAUser === false &&
+        config.twoFactorAuthOn === 'true' &&
+        ipRangeCheck(ipAddress, quantumAddresses) === false
```

Comment:
I'd question the necessity of `=== false`. If we're having issues with these not being Boolean I'd suggest that code needs looking at.

Reply:
"I'd suggest that code needs looking at" I agree.